jupyter_scheduler/handlers.py

```diff
@@ -163,16 +164,16 @@ async def post(self):
     async def patch(self, job_id):
         payload = self.get_json_body()
 
-        if "status" not in payload:
-            raise tornado.web.HTTPError(500, "Field 'status' missing in request body")
+        status = payload.get("status", None)
+        status = Status(status) if status else None
+
+        if status and status != Status.STOPPED:
+            raise tornado.web.HTTPError(500, "Value of 'STOPPED' only allowed for field 'status'")
```

Comment:
This is confusing; it suggests that you've supplied a value 'STOPPED' for a field that is not 'status'

```suggestion
            raise tornado.web.HTTPError(500, "Field 'status' can be set to 'STOPPED' only")
```